packages/app/src/cli/services/payments/extension-to-toml.ts

```diff
@@ -47,31 +51,32 @@ export enum DashboardPaymentExtensionType {
 }
 
 export function buildTomlObject(extension: ExtensionRegistration): string {
-  switch (extension.type) {
-    case DashboardPaymentExtensionType.Offsite:
+  const context = extension.activeVersion?.context || extension.draftVersion?.context || typeToContext(extension.type)
```

Comment:
In the current tests we have in packages/app/src/cli/services/payments/extension-to-toml.test.ts, we don't have a case where we have the context in registration I believe. You may need to add a spec for the CLI scenario